server/app/tool/src/api.rs

```diff
@@ -0,0 +1,124 @@
+    .subcommand(SubCommand::with_name("set-policy")
+        .about("Set the <TEE> specific evaluation `Policy(.regp)` according to the content of <POLICY>.")
+        .arg(Arg::from_usage("--addr=[ADDR] 'Attestation Service's `management-sock` address.'"))
+        .arg(Arg::from_usage("--tee=[TEE] 'The target TEE name'"))
+        .arg(Arg::from_usage("--policy=[POLICY] 'Policy path'"))
+    )
+    .subcommand(SubCommand::with_name("set-reference-data")
+        .about("Set the <TEE> specific evaluation `Reference Data(.json)` according to the content of <REFERENCE_DATA>.")
+        .arg(Arg::from_usage("--addr=[ADDR] 'Attestation Service's `management-sock` address.'"))
+        .arg(Arg::from_usage("--tee=[TEE] 'The target TEE name'"))
+        .arg(Arg::from_usage("--reference-data=[REFERENCE_DATA] 'Reference Data path'"))
+    )
+    .subcommand(SubCommand::with_name("get-policy")
+        .about("Get the <TEE> specific evaluation `Policy(.regp)` from Attestation Server.")
+        .arg(Arg::from_usage("--addr=[ADDR] 'Attestation Service's `management-sock` address.'"))
+        .arg(Arg::from_usage("--tee=[TEE] 'The target TEE name'"))
+    )
+    .subcommand(SubCommand::with_name("get-reference-data")
+        .about("Get the <TEE> specific `Reference Data(.json)` from Attestation Server.")
+        .arg(Arg::from_usage("--addr=[ADDR] 'Attestation Service's `management-sock` address.'"))
+        .arg(Arg::from_usage("--tee=[TEE] 'The target TEE name'"))
+    )
+    .subcommand(SubCommand::with_name("restore-default-policy")
+        .about("Restore the Attestation Server's <TEE> specific `Policy(.rego)` to default.")
+        .arg(Arg::from_usage("--addr=[ADDR] 'Attestation Service's `management-sock` address.'"))
+        .arg(Arg::from_usage("--tee=[TEE] 'The target TEE name'"))
+    )
+    .subcommand(SubCommand::with_name("restore-default-reference-data")
+        .about("Restore the Attestation Server's <TEE> specific `Reference Data(.json)` to default.")
+        .arg(Arg::from_usage("--addr=[ADDR] 'Attestation Service's `management-sock` address.'"))
+        .arg(Arg::from_usage("--tee=[TEE] 'The target TEE name'"))
+    )
+    .subcommand(SubCommand::with_name("test-attestation")
+        .about("Test Attestation Server's `attestation` function with the input <EVIDENCE>")
+        .arg(Arg::from_usage("--addr=[ADDR] 'Attestation Service's `attestation-sock` address.'"))
+        .arg(Arg::from_usage("--evidence=[EVIDENCE] 'The evidence which is evaluated by Attestation Server. Default is a imbedded `sample` TEE's evidence'"))
+    )
+    .get_matches();
+
+    if let Some(matches) = matches.subcommand_matches("get-policy") {
+        management::get_policy_cmd(matches.value_of("tee"), matches.value_of("addr")).await?;
+    }
+
+    if let Some(matches) = matches.subcommand_matches("get-reference-data") {
+        management::get_reference_data_cmd(matches.value_of("tee"), matches.value_of("addr"))
+            .await?;
+    }
```

Comment:
Instead of 6 if statements, can this be arranged as one match block, like [this example of subcommand_name](https://docs.rs/clap/latest/clap/struct.ArgMatches.html#method.subcommand_name)?